packages/client/src/settings/types/configuration.ts

```diff
@@ -1,27 +1,32 @@
-export enum SchemaFieldType {
+export enum ConfigurationSchemaFieldType {
   Number = 'number',
   Boolean = 'boolean',
   String = 'string',
   Object = 'object',
-  PreOwned = 'PreOwned',
 }
 
+export type ConfigurationSecurity = {
+  resources: Array<{
+    name: string;
+    action: string;
+  }>;
+  scopes: string[];
+};
+
+export type ConfigurationProperty = {
+  code: string;
+  description: string;
+  value: string;
+  schemaFieldType: ConfigurationSchemaFieldType;
+  security: ConfigurationSecurity;
+  contextValues?: [{ code: string; value: string }];
```

Comment:
Can you change this to  `Array<{ code: string; value: string }>` or `{ code: string; value: string }[]` , for consistency? It's how we are doing in the project.

Reply:
changed to `Array<{ code: string; value: string }>`